src/makeRobot.js

```diff
@@ -38,7 +38,70 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
When `variable name === prop name` you can use shortened prop application:
```suggestion
    name,
    wheels,
    version,
```